Lack of ARM64 support?

There is no docker image for arm64, pulling it on an arm64 machine just runs the amd64 binary, causing it to fail due to an exec format error.

```js
go version go1.23.1 linux/arm64
AlmaLinux 9.4 hexcontainer.aarch64 #1 SMP PREEMPT_DYNAMIC Tue Nov 7 20:09:26 UTC 2023 aarch64 aarch64 aarch64 GNU/Linux
```

Cloning the current repo tree and running DiceDB via `go run main.go` gives me the following output:
```go
# github.com/dicedb/dice/internal/eval
internal/eval/eval_linux_arm64.go:28:5: undefined: config
internal/eval/eval_linux_arm64.go:29:3: nil is not used
```

Does this mean that DiceDB currently doesn't support arm64? If yes, will it support arm64 in the future? :)

--- a/internal/eval/eval_linux_arm64.go
+++ b/internal/eval/eval_linux_arm64.go
@@ -5,6 +5,7 @@ package eval
 import (
 	"syscall"
 
+	"github.com/dicedb/dice/config"
 	"github.com/dicedb/dice/internal/clientio"
 	diceerrors "github.com/dicedb/dice/internal/errors"
 	"github.com/dicedb/dice/internal/server/utils"
@@ -26,7 +27,7 @@ func EvalBGREWRITEAOF(args []string, store *dstore.Store) []byte {
 	// TODO: Each child process would now have a copy of the network file descriptor thus resulting in resource leaks.
 	// TODO: We need to find an alternative approach for the multi-threaded environment.
 	if config.EnableMultiThreading {
-		nil
+		return nil
 	}
 	childThreadID, _, _ := syscall.Syscall(syscall.SYS_GETTID, 0, 0, 0)
 	newChild, _, _ := syscall.Syscall(syscall.SYS_CLONE, syscall.CLONE_PARENT_SETTID|syscall.CLONE_CHILD_CLEARTID|uintptr(syscall.SIGCHLD), 0, childThreadID)

